pkgs/applications/editors/vim/plugins/nvim-treesitter/update-shell.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import ../../../../../.. { } }:
```

Comment:
Please package `update.py` with Nix using `buildPythonApplication` or simply `writeScriptBin`, instead of a mkShell shell. And then we can define `nvim-treesitter.passthru.updateScript` to allow simpler auto-updating.

Reply:
imo just running the script here is simpler as opposed to having to `nix run` and "compie" the script. this is also what the vim update script is doing atm